README.md

```diff
@@ -1 +1,32 @@
-# node-modules-core-lab
\ No newline at end of file
+# Importing fs, url and path
+
+## Objectives
+
+1. Write a script to import fs, url and path
+1. Invoke methods from core modules fs, url and path
+1. Output results
+
+## Introduction
+
+In this lab, you'll start mastering the three core modules: fs, url and path.
+
+## Instructions
+
+1. Create a program file `url-transformer.js`
```

Comment:
give them the file they'll be coding in. you never where they'll create files and it could effect the tests